Gemfile

```diff
@@ -131,6 +133,8 @@ source "https://rails-assets.org" do
   gem "rails-assets-angular-mocks",                   "~>1.5.0"
   gem "rails-assets-angular-patternfly-sass",         "~>3.3.2"
   gem "rails-assets-angular-sanitize",                "~>1.5.0"
+  gem 'rails-assets-ui-router',                       "~>0.2.18"
+  gem "rails-assets-bootstrap-datepicker",            "~>1.4.0"
```

Comment:
shouldn't need `bootstrap-datepicker`, `patternfly-sass` already depends on it.. (and I don't see it used anywhere in this pr?)